src/content/security/cluster-access/index.md

```diff
@@ -38,32 +39,32 @@ If you would like to know more about the different parts of the Giant Swarm infr
 
 ## User access
 
-User access is limited to the offered APIs for interaction with your clusters.
+User access is provided via two APIs for interaction with your clusters.
 
 ### Giant Swarm API
 
 High-level cluster management access is provided to you via the Giant Swarm API. This includes creating, scaling, and deleting your clusters, as well as other organization and user management functions.
-Network access to the API endpoint is usually whitelisted to a certain range of IP addresses. It can also be configured to work over a VPN following the general VPN connection schema shown below under [admin access](#admin-access). In this case the connection to the API residing in the cluster can be established only via your configured VPN.
+Network access to the API endpoint is typically whitelisted to a certain range of IP addresses. It can also be configured to work over a Virtual Private Network (VPN) following the general VPN connection schema shown below under [admin access](#admin-access). In this case, the connection to the API residing in the cluster can be established only via your configured VPN.
```

Comment:
Can `virtual private network` be spelled in lowercase? Title case makes it look like some product name.

Unrelated to your changes, but can we find a replacement term for `whitelisted` here, too?